features/step_definitions/basic_steps.rb

```diff
@@ -28,6 +28,10 @@
   }
 end
 
+Then("I fill in {string}") do |content|
+  fill_in with: content 
```

Comment:
`fill_in` what? What field are you filling in? The way this step is defined you might get a false positive.

Reply:
We have now removed it since it was unnecessary and we fixed the test accordingly.